index/miner/module/meta.go

```diff
@@ -18,51 +18,53 @@ import (
 )
```

Comment:
All changes in the indexes aren't that interesting. I'm simplifying some logic and having a bit more ordered logic since I did some exploration about some situations... then took advantage and improved things a bit.